src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuPurapServiceImpl.java

```diff
@@ -253,11 +253,18 @@ public KualiDecimal getApoLimit(final PurchasingDocument purchasingDocument) {
         }
 
         if (ObjectUtils.isNull(purchaseOrderTotalLimit)) {
-            String defaultLimitParameterName = shouldUseFederalApoLimit(purchasingDocument)
-                    ? CUPurapParameterConstants.AUTOMATIC_FEDERAL_PURCHASE_ORDER_DEFAULT_LIMIT_AMOUNT
-                    : PurapParameterConstants.AUTOMATIC_PURCHASE_ORDER_DEFAULT_LIMIT_AMOUNT;
-            
-            final String defaultLimit = parameterService.getParameterValueAsString(RequisitionDocument.class, defaultLimitParameterName);
+            String defaultLimit = "";
+
+            if (shouldUseFederalApoLimit(purchasingDocument)) {
+                defaultLimit = parameterService.getParameterValueAsString(RequisitionDocument.class, CUPurapParameterConstants.AUTOMATIC_FEDERAL_PURCHASE_ORDER_DEFAULT_LIMIT_AMOUNT);
```

Comment:
Since the related base `AUTOMATIC_PURCHASE_ORDER_DEFAULT_LIMIT_AMOUNT` parameter is being renamed to `APO_LIMIT` with a `PurchaseOrder` component, we may want to consider renaming this CU-specific parameter to follow a similar convention. It might be worth asking the functionals if they think such a change to our customization makes sense.

Reply:
I saw Cathy's comment on the user story, where it sounds like she would prefer to keep the parameter name as-is if there are no objections from other functionals. Given that, I'm fine with leaving this custom parameter name as-is.